frontend/projects/upgrade/src/app/core/segments/store/segments.model.ts

```diff
@@ -225,8 +225,15 @@ export interface SegmentState extends EntityState<Segment> {
   sortAs: SORT_AS_DIRECTION;
 }
 
+export interface GlobalSegmentState extends EntityState<Segment> {
```

Comment:
This is interesting. It's not exactly a pattern we've used, having 2 feature-level states and reducers that share the same ngrx files. I think it makes sense though, it's separate enough in that it's loaded up and presented independently, but there's obvious major overlap and I don't think anyone's going to get confused. I guess I have nothing productive to say, just needed to type and talk it out, looks good 👍